src/main/java/Board.java

```diff
@@ -133,14 +137,17 @@ public String toString() {
                 }
             }
         }
-        sb.append("  A  B  C  D  E  F  G  H");
+        sb.append(" ");
+        for (int j = 0; j < numberOfColumns; j++) {
+            sb.append("  ").append((char) ('A' + j));
+        }
         return sb.toString();
     }
 
     @NotNull
     @Override
-    public Iterator<Map.Entry<Position, Cell>> iterator() {
```

Comment:
We need also the color of the stone when iterating

Reply:
The color of the stone can be obtained from the position if needed